src/renderer/columnRenderers/colFileName.tsx

```diff
@@ -11,27 +10,30 @@ interface IFileNameProps {
 export const ColFileName = ( props: IFileNameProps ) =>  {
 	const { dataArgs } = props;
 
+	const formatFileName = () => {
+		return dataArgs.cellData.replace(/^.*\/app\/public\/wp-content/g, 'wp-content');
+	}
+
 	const getSelectedCount = () => {
 		return dataArgs.data.filter(
 			data => data.isChecked
 		).length;
 	}
 
- 	const formatFileName = () => {
-		return dataArgs.cellData.replace(/^.*\/app\/public\/wp-content/g, `.${path.sep}wp-content`);
-	}
-
 	if (dataArgs.isHeader) {
 		return(
 			<div className='fileList_File_Name_Header'>
 				{props.optimizationStatus === OptimizerStatus.BEFORE ? (getSelectedCount() + ' Images Selected') : 'File Name'}
 			</div>
 		);
 	} else {
-		return(
-			<div className='fileList_File_Name_Row'>
+		return (
+			<div
+				className='fileList_File_Name_Row'
+				title={formatFileName()}
```

Comment:
A small optimization/maintainability request: could you capture this in a variable and use that computed value in the `title` attribute as well as the `div` contents?

Short, abbreviate example of what I mean:

```
const fileName = dataArgs.cellData.replace(/^.*\/app\/public\/wp-content/g, 'wp-content');

// ... 

return (
    <div title={fileName}>{fileName}</div>
);
```

Reply:
✅

Reply:
:thankee: